clients/shared/table_config.go

```diff
@@ -15,50 +15,30 @@ import (
 	"github.com/artie-labs/transfer/lib/typing/columns"
 )
 
+func isCommentSet(comment string) bool {
+	return !(comment == "" || comment == "<nil>")
+}
+
 type GetTableCfgArgs struct {
 	Dwh       destination.DataWarehouse
 	TableID   sql.TableIdentifier
 	ConfigMap *types.DwhToTablesConfigMap
 	Query     string
 	Args      []any
-	// Name of the column
-	ColumnNameLabel string
-	// Column type
-	ColumnTypeLabel string
-	// Description of the column (used to annotate whether we need to backfill or not)
-	ColumnDescLabel    string
-	EmptyCommentValue  *string
-	DropDeletedColumns bool
-}
 
-func (g GetTableCfgArgs) ShouldParseComment(comment string) bool {
-	if g.EmptyCommentValue != nil && comment == *g.EmptyCommentValue {
-		return false
-	}
-
-	// Snowflake and Redshift both will return `<nil>` if the comment does not exist, this will check the value.
-	// BigQuery returns ""
-	return true
+	ColumnNameForName        string
```

Comment:
@nathan-artie I renamed these to be more clear

Reply:
For MSSQL though `ColumnNameForDescription` is misleading because it's not the column name used for the description like the other data sources, it's the column name that stores the default value. Unless you're thinking of the default value as a type of description?

I also find it somewhat confusing later on when we do:
```
if comment, isOk := row[g.ColumnNameForDescription]; isOk && isCommentSet(comment) {
``` 
Because the code leads me to believe that this column is storing a comment, but for MSSQL we're storing the default value not a comment.

Reply:
For OLAP sources, we're relying on the comment column to know whether this column has been backfilled.

OLTP sources have a concept of default values, which we're getting the data from as you can see from the SQL query

Reply:
I would say we should just have two different fields, a `ColumnNameForComment` and a `ColumnNameForDefaultValue`.

Reply:
Sure